### PR TITLE
[PRP-283] Create staff users

### PR DESCRIPTION
**Pull request by rocketnova**
## Ticket

https://wicmtdp.atlassian.net/browse/PRP-283

- Create an S3 bucket for side-loading data into containers
- Add a script to create staff user records in the participant database using a file in the side-load s3 bucket
- Add a terraform module to create cognito users
- Add a bash script to automate all of the above
- Misc: fix typos

### CODE REVIEW

File: infra/modules/cognito-staff-user/create-staff-users.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+# Variables.
+# The emails get transformed into this format for terraform to ingest:
+# {
+#   "apple@wic.gov": "gallatin",
+#   "carrot@wic.gov": "gallatin",
+#   "eggplant@wic.gov": "gallatin",
+#   "banana@wic.gov": "missoula",
+#   "daikon@wic.gov": "missoula"
+# }
```
Comment (microwavenby) [inline infra/modules/cognito-staff-user/create-staff-users.sh:46]: This is _not a request for a change_ but I would ask why have two file formats here?
Why not have the one we need for terraform from the beginning?

File: infra/modules/cognito-staff-user/create-staff-users.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  -var="user_emails=${TRANSFORMED_USER_EMAILS}"
+
+# Save output to s3.
+# The ECS Task process needs a file that maps Cognito UUIDs to local agencies. This is automatically created and
+# temporarily saved locally. It will be cleaned up as the last step.
+terraform output -json | jq .staff_user_list.value > $UUID_FILENAME
+aws s3api put-object \
+  --bucket $BUCKET_NAME \
+  --key "seed/${UUID_FILENAME}" \
+  --body $UUID_FILENAME
```
Comment (microwavenby) [inline infra/modules/cognito-staff-user/create-staff-users.sh:70]: I think technically there's a way to pipe this directly to S3, but. 🤷🏻‍♀️

Comment (microwavenby) [reply infra/modules/cognito-staff-user/create-staff-users.sh:70]: Yeah... you could do this at the cost of debuggability:

`terraform output -json | jq .staff_user_list.value | aws s3 cp - s3://${BUCKET_NAME}/seed/${UUID_FILENAME}`

https://loige.co/aws-command-line-s3-content-from-stdin-or-to-stdout/#the-magic---option-in-aws-cp

**Review by microwavenby — APPROVED** [review]

File: infra/modules/cognito-staff-user/create-staff-users.sh (hunk 1 above)
Comment (rocketnova) [reply infra/modules/cognito-staff-user/create-staff-users.sh:46]: I did originally, but thought that would be _more_ confusing to other people less familiar with the system? 🤷 Do you think _this_ is more confusing? I have completely lost the ability to discern that

File: infra/modules/cognito-staff-user/create-staff-users.sh (hunk 2 above)
Comment (rocketnova) [reply infra/modules/cognito-staff-user/create-staff-users.sh:70]: Oh interesting. 🤔 I'll probably leave it because the debuggability was really useful.

**Pull request merged**